### DISCUSSION TITLE
How to set view_height

### DISCUSSION THREAD

**Discussion by kurt-rhee** [mozman/ezdxf#668] [Q&A] [answered]
I'm wondering if I have many different viewports looking at different objects with different sizes, how do I calculate the view height of the viewport?  

My guess is that there is a viewing angle associated with the `view_center_point` which stays the same and as view height increases, the cone (or rectangle) of view increases.  If this is the case, what is that viewing angle?

Thank you for your help.

**Comment by mozman** [comment, marked as answer]
The example [viewports_in_paperspace.py](https://github.com/mozman/ezdxf/blob/master/examples/viewports_in_paperspace.py) shows how to setup  VIERWPORT entities in paperspace.

![image](https://user-images.githubusercontent.com/4230966/162113538-2d922b5d-9f6c-4d8d-9e0b-ea1865add288.png)

The **drawing add-on does not support viewports,** so you will need an advanced DXF viewer like TrueView or the ODA Viewer or a CAD application to print/export layouts including viewports.
